tools/export/exporters.py

```diff
@@ -167,9 +171,15 @@ def scan_and_copy_resources(self, prj_paths, trg_path, relative=False):
         # Loads the resources into the config system which might expand/modify resources based on config data
         self.resources = config.load_resources(resources)
 
-        # And add the configuration macros to the toolchain
-        self.config_macros = config.get_config_data_macros()
-                
+
+        if hasattr(self, "MBED_CONFIG_HEADER_SUPPORTED") and self.MBED_CONF_ACTIVE :
```

Comment:
'and self.MBED_CONFIG_HEADER_SUPPORTED`. Please do not issue/modify a PR without testing its functionality at least once locally.